src/sentry/api/serializers/models/integration.py

```diff
@@ -47,14 +48,15 @@ def serialize(self, obj, attrs, user):
             # representation.
             pass
         else:
-            data.update({"configOrganization": install.get_organization_config()})
+            if include_config:
```

Comment:
If `obj.get_installation(organization_id=self.organization_id)` has no side-effects, we can just skip it when `include_config` is `False`.